app/models/censuses/census_report.rb

```diff
@@ -0,0 +1,286 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Censuses
+  class CensusReport
+    include ArelHelper
+    include Filter::ControlSections
+
+    def initialize(filter)
+      @filter = filter
+    end
+
+    # what projects should be included?
+    def census_projects_scope
+      scope = GrdaWarehouse::Hud::Project.residential.
+        viewable_by(@filter.user).
+        where(id: @filter.effective_project_ids)
+
+      # Limit ES projects to Night-by-night only by filtering out Entry/Exit ES projects
+      scope = scope.where.not(ProjectType: 1, TrackingMethod: 0) if @filter.limit_es_to_nbn
```

Comment:
This should lean on the `Project.night_by_night` scope, which includes the overrides.  We'll probably need an inverse one to catch entry-exit.

Reply:
Ok. I'm trying to not filter out OTHER project types is "Limit ES to NBN" is selected. For example if you choose Street Outreach and ES as project types, and then choose "Limit ES to NBN," the street outreach projects should still be included. Is that right? I think I can still use the night_by_night somehow.

Reply:
Yeah, it's working correctly, including other project types, but it's not taking into account overrides as the scope does.

Reply:
A coalesce is probably all it needs, but if we can lean on the scope, I think it'll be safer long-term.

Reply:
I feel like there's probably a better way to do this, but heres what I have: https://github.com/greenriver/hmis-warehouse/pull/2628/commits/8f7726034ea213a8aaa13d2a26995845ebf53716

Reply:
Yeah, there probably is, but I think this is better than what was there, and good enough for now.  Thanks!